### DISCUSSION TITLE
download as - file name

### DISCUSSION THREAD

**Discussion by trup** [aclap-dev/video-downloadhelper#1082] [Help and Q&A] [answered]
### Your question

Hello.

Feature request.

When I select "download as", and if the download does not start immediately (for example, when the number of simultaneous downloads is exceeded), the window with the choice of file name will appear only when I am no longer at the computer.

The file name for saving should be requested immediately, after pressing the "download as" button.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: ru
coapp: "checking"
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\AnnA\\Videos",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "concurrent_downloads_max": 5,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "show_context_menu": false,
    "yt_warning": false,
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 19,
    "successfull_dl": 69
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

This was reported once before by another of our fellow users.  You might go search for that discussion.  Meanwhile, as I recall, this issue was fixed in a beta.  You're running Firefox.  Try the VDH beta.  You can find it in the Wiki.  There's a link to the Wiki at the top of every page of this forum.  Maybe that will fix the issue, maybe it won't.  But you should be running the beta anyway.  Everybody should.  The VDH betas contain bug corrections.  They're not experimental versions.  Everybody should be running the beta.

**Comment by paulrouget** [comment, marked as answer]
Fixed in v9.1.

In Firefox beta now, you can try: https://www.downloadhelper.net/beta
